AzureAuth.lutconfig

```diff
@@ -0,0 +1,6 @@
+<LUTConfig Version="1.0">

```

Comment:
If we do want to keep it in source, we should add a comment on the purpose since it wasn't obvious to me immediately.